ui/src/App.tsx

```diff
@@ -1,25 +1,31 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import React from "react";
+import "./App.css";
+import { Layout, Menu, Typography } from "antd";
+import { CompleteProducts } from "./CompleteProducts";
 
-function App() {
+const { Title } = Typography;
+const { Header, Footer, Content } = Layout;
+
+function App(): JSX.Element {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.tsx</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
+    <Layout className={"App"}>
+      <Header className={"App-header"}>
+        <Menu
+          className={"App-header-menu"}
+          mode="horizontal"
+          defaultSelectedKeys={["home"]}
         >
-          Learn React
-        </a>
-      </header>
-    </div>
+          <Menu.Item key="home">Home</Menu.Item>
```

Comment:
bleee plaintext :P jako jedno z kolejnych zadanego to musimy dodać obsługę językową. Jak nie zrobimy na początku to już nie robimy. dodam zadanko

Reply:
Agree, but for the first commit I think it's fine 😅

Reply:
ok. And task https://trello.com/c/4EZQN4jo/10-wprowadzi%C4%87-obs%C5%82ug%C4%99-r%C3%B3%C5%BCnych-j%C4%99zyk%C3%B3w-i18n